src/Components/Server/test/Circuits/ServerComponentDeserializerTest.cs

```diff
@@ -8,6 +8,11 @@
 
 namespace Microsoft.AspNetCore.Components.Server.Circuits;
 
+// TODO:
+// - Test that the marker sequence is required.
+// - Test that the marker sequence must match the descriptor sequence.
+// - Test that markers cannot be duplicated.
```

Comment:
I will implement these after settling on the approach for registering dynamicaly-SSR'd components with .NET.